### PR TITLE
feat: billing cycle date

### PR DESCRIPTION
**Pull request by ayusham001**
/claim #337

### CODE REVIEW

File: android/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,7 @@
+#Fri Sep 08 21:01:35 IST 2023
+networkTimeout=10000
 distributionBase=GRADLE_USER_HOME
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.2-bin.zip
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.0.2-all.zip
-networkTimeout=10000
-zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
```
Comment (rishi-raj-jain) [inline android/gradle/wrapper/gradle-wrapper.properties:7]: Why these changes?

Comment (ayusham001) [reply android/gradle/wrapper/gradle-wrapper.properties:7]: Actually, I built it locally, so these changes were made automatically by the command

**Comment by riderx** [comment]
Hello sorry i didn't saw your PR i already accepted other one

**Comment by ayusham001** [comment]
@riderx No problem:)
